### ISSUE TITLE
[libc++][test] libcxx/numerics/c.math/constexpr-cxx23-clang.pass.cpp fails on PowerPC targets

### ISSUE THREAD

**Issue by amy-kwan** [llvm/llvm-project#64426]
When building libc++ running `check-runtimes`, the following issue is exhibited:
```
Command Output (stdout):
--
$ ":" "COMPILED WITH"
$ "/home/amyk/llvm/community/build/bootstrap/stage1/build/./bin/clang++" "/home/amyk/llvm/community/llvm-project/libcxx/test/libcxx/numerics/c.math/constexpr-cxx23-clang.pass.cpp" "-pthread" "--target=powerpc64le-unknown-linux-gnu" "-nostdinc++" "-I" "/home/amyk/llvm/community/build/bootstrap/stage1/build/include/c++/v1" "-I" "/home/amyk/llvm/community/build/bootstrap/stage1/build/include/powerpc64le-unknown-linux-gnu/c++/v1" "-I" "/home/amyk/llvm/community/llvm-project/libcxx/test/support" "-std=c++26" "-Werror" "-Wall" "-Wctad-maybe-unsupported" "-Wextra" "-Wshadow" "-Wundef" "-Wunused-template" "-Wno-unused-command-line-argument" "-Wno-attributes" "-Wno-pessimizing-move" "-Wno-noexcept-type" "-Wno-atomic-alignment" "-Wno-reserved-module-identifier" "-Wno-user-defined-literals" "-Wno-tautological-compare" "-Wsign-compare" "-Wunused-variable" "-Wunused-parameter" "-Wunreachable-code" "-Wno-unused-local-typedef" "-Wno-local-type-template-args" "-Wno-c++11-extensions" "-Wno-unknown-pragmas" "-D_LIBCPP_HAS_NO_PRAGMA_SYSTEM_HEADER" "-D_LIBCPP_ENABLE_EXPERIMENTAL" "-Werror=thread-safety" "-Wuser-defined-warnings" "-lc++experimental" "-nostdlib++" "-L" "/home/amyk/llvm/community/build/bootstrap/stage1/build/./lib/powerpc64le-unknown-linux-gnu" "-Wl,-rpath,/home/amyk/llvm/community/build/bootstrap/stage1/build/./lib/powerpc64le-unknown-linux-gnu" "-lc++" "-o" "/home/amyk/llvm/community/build/bootstrap/stage1/build/runtimes/runtimes-bins/test/libcxx/numerics/c.math/Output/constexpr-cxx23-clang.pass.cpp.dir/t.tmp.exe"
# command stderr:
ExpandFloatResult #0: t16: ppcf128,i32 = ffrexp t14

fatal error: error in backend: Do not know how to expand the result of this operator!
clang++: error: clang frontend command failed with exit code 70 (use -v to see invocation)
```

This issue occurs on both LLVM 17.0.0 rc1 and `main`.
Investigation for this issue required.

**Comment by lei137** [comment]
We currently don't have support for lowering of this builtin.  This didn't show up previously as we don't have a libc++ buildbot.   We always built libc++ for release binaries cause it just worked out of the box.  For V17 since we are missing support we won't be including libc++ in our build binary.  Will target to fix this in main and setup an libc++ buildbot to prevent this in the future.

**Comment by lei137** [comment]
Was thinking something similar to the other PowerPC buildbots.  Don't believe we have a pre-commit builder.  Is that the prefered way?

**Comment by lei137** [comment]
Disabling libcxx tests for now: https://reviews.llvm.org/D158547

**Comment by lei137** [comment]
@mordante [clang-ppc64le-rhel](https://lab.llvm.org/buildbot/#/builders/57) have been updated to also build `libcxx,libcxxabi`.  The plan is still to add a pre-commit CI builder, but that will need to wait till 1Q 2024 as I still need to track down HW to host it.  Just wanted to let you know it's still in the plan.

**Comment by ldionne** [comment]
Fixed by aaa3f72c1ce6e1757df79c0d02e0675201ee07a3
